_licenses/ncsa.txt

```diff
@@ -0,0 +1,65 @@
+---
+title: University of Illinois/NCSA Open Source License
+spdx-id: NCSA
+nickname: UIUC/NCSA
+source: https://opensource.org/licenses/NCSA
+
+description: The University of Illinois/NCSA Open Source License, or UIUC license, is a permissive free software license, based on the <a href="/licenses/mit/">MIT/X11 license</a>  and the <a href="/licenses/bsd-3-clause/">BSD 3-clause License</a>. By combining parts of these two licenses, it attempts to be clearer and more concise than either. It's conditions include requiring the preservation of copyright and license notices both in source and in binary distributions and the prohibtion of using the names of the authors or the project organization to promote or endorse derived products.
```

Comment:
Yes, "based on" seems to be true. I'm only suggesting removing "By combining parts of these two licenses, it attempts to be clearer and more concise than either."